ScoutSuite/output/data/html/partials/azure/services.loggingmonitoring.subscriptions.id.log_alerts.html

```diff
@@ -1,11 +1,18 @@
 <!-- Logging and Monitoring Log Alerts-->
 <script id="services.loggingmonitoring.subscriptions.id.log_alerts.partial" type="text/x-handlebars-template">
     <div id="resource-name" class="list-group-item active">
-        <h4 class="list-group-item-heading">{{name}}</h4>
+        <h4 class="list-group-item-heading">{{@key}}</h4>
```

Comment:
For every parsed resource, we use a unique `id` field as well as a `name`, which is what we display here. Please ensure both values exist for all resources, and use the `name` field at the top of the partial.

Reply:
Changed @key to name and fix issue

Reply:
changed @key for name